appendix/device-checklist.md

```diff
@@ -10,33 +10,35 @@ Before starting, gather a list of all devices to audit that are used for Lullabo
 
 With each device, validate:
 
-- [ ] All computers have a complex password with a screen lock timeout set to no more than 10 minutes.
-- [ ] All computer hard drives are encrypted with FileVault, Bitlocker, or equivalent.
-- [ ] All operating systems and browsers should be patched and up to date.
-- [ ] Confirm VPN app (Proton VPN) is setup and the team member is aware of the "split tunnel" option.
-- [ ] Confirm 1Password is installed with browser extensions.
-- [ ] Confirm an adblocker (uBlock origin, AdGuard) is installed in the default browser.
+- [ ] Confirm all mobile devices have a [screen timeout set to no more than 10 minutes](https://security.lullabot.com/physical/locking.html).
+- [ ] Confirm "Find My" or equivalent is enabled.
+- [ ] All operating systems should be up to date.
+- [ ] All browsers should be up to date.
+- [ ] Confirm [1Password](https://security.lullabot.com/access/password_managers.html) is installed, with browser extensions.
+- [ ] Confirm an adblocker (uBlock origin lite, AdGuard) is installed in the default browser.
 
-## Workstations
+## Computers
 
-- [ ] Confirm "Find My" or equivalent is enabled.
-- [ ] Confirm firewall is enabled.
+- [ ] All computers have a [complex password](https://security.lullabot.com/access/passwords.html), at least 16 characters long. If you add your passwords to 1password, 1password will tell you the strenght of the password; try to get "excellent" strength. Do NOT use guessable patterns like: family names, birthdays, the word “lullabot”, etc.
+- [ ] All computer [hard drives are encrypted](https://security.lullabot.com/physical/hard-drive-encryption.html) with FileVault, Bitlocker, or equivalent.
+- [ ] Confirm firewall is enabled. Document work tasks that require disabling firewall; share with security@lullabot.com.
```

Comment:
What tasks are these? Normally you can allow apps through the firewall without disabling it entirely.

Reply:
Asking people to tell us... gives us a chance to help people not disable the firewall. I'll look and see if I had any notes about task specifics from the last checklist round we did. Some notes mention xdebug, or having a household system when at home, and vpn when away. 

But maybe instead this could be firmer wording.